src/js/rx/containers/Active.jsx

```diff
@@ -26,7 +26,10 @@ class Active extends React.Component {
     this.handleSort = this.handleSort.bind(this);
 
     this.checkWindowSize = _.debounce(() => {
-      if (document.documentElement.clientWidth < 600) {
+      const viewToggleElement = this.refs.viewToggle;
```

Comment:
What's happening here might not be obvious to someone unfamiliar with the behavior, so might be helpful to leave a comment about how the stylesheet is hiding the button when the window's resized to a small screen.